packages/admin/admin/src/dataGrid/MoreActionsMenu.tsx

```diff
@@ -0,0 +1,89 @@
+import { MoreVertical } from "@comet/admin-icons";
+import { Box, Button, Chip, ChipProps, Divider, DividerProps, Menu, MenuList, MenuListProps, MenuProps, Typography } from "@mui/material";
+import * as React from "react";
+import { PropsWithChildren } from "react";
+import { FormattedMessage } from "react-intl";
+
+export function MoreActionsDivider(props: DividerProps) {
+    return <Divider sx={{ margin: "8px 0", borderColor: (theme) => theme.palette.grey[50] }} {...props} />;
+}
+
+interface MoreActionsGroupProps {
+    groupTitle: React.ReactNode;
+    menuListProps?: MenuListProps;
+    typographyProps?: React.ComponentProps<typeof Typography>;
+}
+
+export function MoreActionsGroup({ groupTitle, children, menuListProps, typographyProps }: PropsWithChildren<MoreActionsGroupProps>) {
+    return (
+        <>
+            <Typography variant="subtitle2" color={(theme) => theme.palette.grey[500]} fontWeight="bold" mt={4} px={1} {...typographyProps}>
+                {groupTitle}
+            </Typography>
+            <MenuList {...menuListProps}>{children}</MenuList>
+        </>
+    );
+}
+
+interface MoreActionsMenuProps {
+    selectionSize?: number;
+    buttonProps?: React.ComponentProps<typeof Button>;
+    menuProps?: Partial<MenuProps>;
+    chipProps?: Partial<ChipProps>;
+    children?: (props: { handleClose: () => void; selectedItemsChip: React.ComponentType<Partial<ChipProps>> }) => React.ReactNode;
```

Comment:
Why do you pass `selectedItemsChip` to the render function?

Reply:
To have the styled chip component  🤷🏻‍♂️

Reply:
But couldn't it be a "normal" component that can be imported, an no render prop?

Reply:
@thomasdax98 wanted me to do it like this. Check the comments i linked above